### PR TITLE
update nobooks

### PR DESCRIPTION
**Pull request by tchaton**
## What does this PR do?

This PR updates notebooks:
- [x] Add Colab Link
- [x] Add Introduction to notebook
- [x] 1 cell for each code block
- [x] 1 comment every cell
- [x] Add install
- [x] Add greetings     
- [x] In predict, add link to `fine-tune colab`
- [x] Added to CHANGELOG

Here is how it looks. 

Screenshot 2021-01-29 at 09 37 46

Fixes # (issue)

## Before submitting
- [x] Was this discussed/approved via a Github issue? (no need for typos and docs improvements)
- [x] Did you read the [contributor guideline](https://github.com/PyTorchLightning/pytorch-lightning-lash/blob/master/.github/CONTRIBUTING.md), Pull Request section?
- [x] Did you make sure your PR does only one thing, instead of bundling different changes together?
- [x] Did you make sure to update the documentation with your changes?
- [x] Did you write any new necessary tests? [not needed for typos/docs]
- [x] Did you verify new and existing tests pass locally with your changes?
- [x] If you made a notable change (that affects users), did you update the [CHANGELOG](https://github.com/PyTorchLightning/lightning-flash/blob/master/CHANGELOG.md)?

## PR review
 - [x] Is this pull request ready for review? (if not, please submit in draft mode)

Anyone in the community is free to review the PR once the tests have passed.
If we didn't discuss your PR in Github issues there's a high chance it will not be merged.

## Did you have fun?
Make sure you had fun coding 🙃

### CODE REVIEW

**Comment by Borda** [comment]
how is it related to #3?

File: flash_notebooks/finetuning/image_classification.ipynb [hunk 1]
```diff
@@ -0,0 +1,241 @@
+{
```
Comment (edenlightning) [inline flash_notebooks/finetuning/image_classification.ipynb:1]: <p>Add: to classify images of ants and bees</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/PyTorchLightning/lightning-flash/pull/9/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Comment (edenlightning) [inline flash_notebooks/finetuning/image_classification.ipynb:1]: <p> </p><p>Maybe</p><p>In this example we will be downloading the Hymenoptera dataset from a URL, and save that in a 'data' directory.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/PyTorchLightning/lightning-flash/pull/9/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Comment (edenlightning) [inline flash_notebooks/finetuning/image_classification.ipynb:1]: <p>Flash Tasks have built-in DataModules that you can abuse to organize your data. Pass in a train, validation and test folders and Flash will take care of the rest.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/PyTorchLightning/lightning-flash/pull/9/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Comment (edenlightning) [inline flash_notebooks/finetuning/image_classification.ipynb:1]: <p>Create the <code>ImageClassifier</code>  task. By default, the ImageClassifier task uses a resnet-18 backbone to train or finetune your model.</p><p> </p><p>quick question: don't you need to specify num_classes in the datamodule?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/PyTorchLightning/lightning-flash/pull/9/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Comment (edenlightning) [inline flash_notebooks/finetuning/image_classification.ipynb:1]: <p>The trainer object can be used for training or fine-tuning tasks on new sets of data. </p><p> </p><p>You can pass in parameters to control the training routine- limit the number of epochs, run on GPUs or TPUs, etc.</p><p>For more details, read the Trainer documentation (add link).</p><p> </p><p>In this demo, we will limit the fine-tuning to run just one epoch using <code>max_epochs=1</code>.</p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/PyTorchLightning/lightning-flash/pull/9/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Comment (edenlightning) [inline flash_notebooks/finetuning/image_classification.ipynb:1]: <p>change title to "Finetune the model".</p><p> </p><p>Explain why you use unfreeze_milestones,</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/PyTorchLightning/lightning-flash/pull/9/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Comment (edenlightning) [inline flash_notebooks/finetuning/image_classification.ipynb:1]: <p>Add predict examples? Maybe one of a bee and one of an ant?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/PyTorchLightning/lightning-flash/pull/9/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Comment (edenlightning) [inline flash_notebooks/finetuning/image_classification.ipynb:1]: <p>Maybe use flash logo?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/PyTorchLightning/lightning-flash/pull/9/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Comment (tchaton) [reply flash_notebooks/finetuning/image_classification.ipynb:1]: <p>Done !</p>

Comment (tchaton) [reply flash_notebooks/finetuning/image_classification.ipynb:1]: <p>Done !</p>

Comment (tchaton) [reply flash_notebooks/finetuning/image_classification.ipynb:1]: <p>Done !</p>

**Comment by teddykoker** [comment]
Think we should add `treon` back here: https://github.com/PyTorchLightning/lightning-flash/blob/update_notebooks/.github/workflows/ci-notebook.yml#L49-L52 (no need to run pytest on the `notebooks/` directory, we have the e2e tests now)

File: flash_notebooks/finetuning/image_classification.ipynb (hunk 1 above)
Comment (tchaton) [reply flash_notebooks/finetuning/image_classification.ipynb:1]: It is inferred automatically.

**Review by teddykoker — APPROVED** [review]

**Pull request merged**